tests/test_resolver.py

```diff
@@ -21,7 +21,7 @@
 # This test can be removed once the logging transforms, alchemy like,
 # into real functionality.
 def test_open_stage_and_logging(capfd):
-    Usd.Stage.Open("resources/empty_shot.usda")
+    open_stage("resources/empty_shot.usda")
```

Comment:
[consider] As discussed offline, would be good to pull this out as a separate commit, since it addresses a bug in a previous PR.

Reply:
Im happy that it's in a separate commit in this PR (the granularity of the other changes would be more worthy of effort than PR juggling here imho), but we shouldn't hold up progress on the functional aspects, as long as we have `plan` for the main thing we want to improve for an MVP

Reply:
Sorry my bad, it is indeed already in a separate commit :facepalm: